packages/vuetify/src/locale/sv.ts

```diff
@@ -0,0 +1,39 @@
+export default {
+  close: 'Stäng',
+  dataIterator: {
+    pageText: '{0}-{1} av {2}',
+    noResultsText: 'Inga poster funna',
+    loadingText: 'Laddar data...',
+  },
+  dataTable: {
+    itemsPerPageText: 'Rader per sida:',
+    ariaLabel: {
+      sortDescending: ': Sorterat fallande. Aktivera för att ta bort sortering.',
+      sortAscending: ': Sorterat stigande. Aktivera för sortering fallande.',
+      sortNone: ': Osorterat. Aktivera för sortering stigande.',
+    },
+  },
+  dataFooter: {
+    itemsPerPageText: 'Rader per sida:',
```

Comment:
v-data-footer can be used outside of a table context (e.g. it's used for v-data-iterator), so it should probably not be `Rader` here

Reply:
Ok, there is no perfect single translation for "items" in Swedish but maybe "Objekt per sida"  is better.

Reply:
"objekt" works. Maybe "poster". Of the two I think I prefer objekt.